### ISSUE TITLE
Can't configure mosh ports

### ISSUE THREAD

**Issue by magical** [rpwoodbu/mosh-chrome#120]
I've been using a non-standard port range for mosh on one of my servers (60100-60199 instead of 60000-60099). To connect with the command-line version of mosh, i simply run `mosh -p 60100:60199 server`; however, mosh-chrome does not seem to have any way to configure the port range. (You can give a port when connecting in manual mode, but that's not really the same.)

**Comment by rpwoodbu** [comment]
Yeah, this would be a good enhancement. You can get the effect by specifying the Mosh server command, like so: `/usr/bin/mosh-server -p 60100:60199`

You may find you need to add other options when doing it that way, though, as there are some defaults around localization and whatnot that will get omitted. All the more reason that this should be offered in a more straightforward manner.

**Comment by magical** [comment]
Aha, thanks. I didn't realize it was just passed along to the server like that. That'll work for now.

Here's the server command i ended up using, for anyone with the same problem: `/usr/bin/mosh-server new -s -c 256 -p 60100:60199 -l LANG=en_US.UTF-8`
